Desktop version

mat-levch opened the pull request:
Адаптація під десктоп.
Задав властивості для контейнеру від 1280px
@media screen and (min-width: 1280px) {
    width: 1280px;
  }

mat-levch committed:
Desktop version

Адаптація під десктоп.
Задав властивості для контейнеру від 1280px
@media screen and (min-width: 1280px) {
    width: 1280px;
  }

mat-levch committed:
Update about.html

YanaKarpliuk committed:
Merge branch 'feature/about' of https://github.com/YanaKarpliuk/icecream-team-11 into feature/about

mat-levch committed:
Milk edits

зафіксував молоко при ширині 1280px

mat-levch committed:
Modal about

Перша версія модального вікна секції about. Планую додати деякі елементи

YanaKarpliuk reviewed (changes requested):
Все інше ок. З модалкою є деякі проблеми.
При ширині 320 кнопка налазить на текст - https://prnt.sc/aupEkJqMOiuD 
Також коли модалка відкрита, сам сайт не має скролитися (ми так робити із студією)

YanaKarpliuk committed:
Merge branch 'feature/about' of https://github.com/YanaKarpliuk/icecream-team-11 into feature/about

mat-levch committed:
Edits after team-lead remarks

Виправив проблему з іконкою модального вікна при 320px, задав transition для закриття та відкриття, а також зафіксував модальне вікно у відкритому стані.

YanaKarpliuk committed:
Merge branch 'feature/about' of https://github.com/YanaKarpliuk/icecream-team-11 into feature/about

YanaKarpliuk committed:
Merge branch 'main' into feature/about
